torch/distributed/pipeline/sync/checkpoint.py

```diff
@@ -167,8 +168,10 @@ def is_checkpointing() -> bool:
 
 
 def is_recomputing() -> bool:
-    """Whether the current forward propagation is under checkpoint
-    recomputation. Use this to prevent duplicated side-effects at forward
+    """
+    Whether the current forward propagation is under checkpoint recomputation.
+
+    Use this to prevent duplicated side-effects at forward
```

Comment:
```suggestion
    """Determine whether the current forward propagation is under checkpoint recomputation.

    Use this function to prevent duplicated side-effects at forward
```